api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -130,4 +131,13 @@ public void setFirstSignInTime(Timestamp firstSignInTime) {
   public void setAuthorities(Set<Authority> newAuthorities) {
     this.authorities = newAuthorities;
   }
+
+  @OneToMany(fetch = FetchType.EAGER, mappedBy = "user")
```

Comment:
I believe `EAGER` doesn't actually work; I'm not sure if there's a workaround. So, watch out for cases where you get a user in a session, and then try to use the roles after the session is closed (which will cause an error).

Reply:
@markfickett I was fairly sure eager *does* work, it just does it with a second query instead of a join. (Which seems appropriate here.)

I think maybe what you were thinking of is that JoinType.FETCH doesn't work with JPA:

https://stackoverflow.com/questions/463349/jpa-eager-fetch-does-not-join

Does that sound right?

Reply:
I'll believe it.